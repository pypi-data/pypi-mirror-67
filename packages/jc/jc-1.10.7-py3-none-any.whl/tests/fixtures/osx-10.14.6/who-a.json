[{"event": "reboot", "time": "Feb 7 23:31", "pid": 1}, {"user": "kbrazil", "tty": "console", "time": "Feb 7 23:32", "idle": "old", "pid": 105}, {"user": "kbrazil", "tty": "ttys000", "time": "Feb 13 16:44", "idle": "00:02", "pid": 51217, "comment": "term=0 exit=0"}, {"user": "kbrazil", "tty": "ttys001", "time": "Feb 11 21:25", "idle": "00:02", "pid": 81390}, {"user": "kbrazil", "tty": "ttys002", "time": "Mar 1 15:18", "idle": ".", "pid": 9574}, {"user": "kbrazil", "tty": "ttys003", "time": "Feb 28 08:59", "idle": "01:06", "pid": 41402}, {"user": "kbrazil", "tty": "ttys004", "time": "Mar 1 17:11", "idle": ".", "pid": 15679, "comment": "term=0 exit=0"}]
