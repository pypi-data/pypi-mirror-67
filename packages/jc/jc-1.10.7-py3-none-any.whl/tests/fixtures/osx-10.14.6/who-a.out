USER     LINE     WHEN         IDLE  	   PID	COMMENT
reboot   ~        Feb  7 23:31   .   	     1
kbrazil  console  Feb  7 23:32  old  	   105
kbrazil  ttys000  Feb 13 16:44 00:02 	 51217	term=0 exit=0
kbrazil  ttys001  Feb 11 21:25 00:02 	 81390
kbrazil  ttys002  Mar  1 15:18   .   	  9574
kbrazil  ttys003  Feb 28 08:59 01:06 	 41402
kbrazil  ttys004  Mar  1 17:11   .   	 15679	term=0 exit=0
   .       run-level 3
